### PR TITLE
Sarinali/cfd 67 comment component

### PR DESCRIPTION
**Pull request by sarinali**
### Description
Created comment component on expo app

Completes CFD-67

- [x] New feature (non-breaking change which adds functionality)

IMG_5746

Test:

/src/apps/expo/app/index.tsx:

```import type { ImageSourcePropType } from "react-native";";

const imageUrl = 'https://static.wikia.nocookie.net/acc-official-database/images/9/91/El_gato.jpg/revision/latest?cb=20220709001857';
const imageSourceProp: ImageSourcePropType = { uri: imageUrl };
<Comment user="Sarina" user_pic={imageSourceProp} time_elapsed="10h" comment_body="thank you so much for merging my PR so quickly i love my PLs and they are awesome #ilovebakerandganesh"></Comment>```

### CODE REVIEW

File: apps/expo/src/app/index.tsx [hunk 1]
```diff
@@ -6,6 +6,9 @@ import { Stack } from "expo-router";
 
 import registerForPushNotificationsAsync from "~/notifications/registerNotifications";
 
+// const imageUrl = 'https://static.wikia.nocookie.net/acc-official-database/images/9/91/El_gato.jpg/revision/latest?cb=20220709001857';
+// const imageSourceProp: ImageSourcePropType = { uri: imageUrl };
```
Comment (RyanL123) [inline apps/expo/src/app/index.tsx:10]: Let's not leave in commented code for better style. If you need to save these you can put them in the ticket description

Comment (sarinali) [reply apps/expo/src/app/index.tsx]: just deleted!

**Review by RyanL123 — DISMISSED** [review]
LGTM, thanks!

**Comment by BakerWJ** [comment]
why are there 28 files being changed in this pr

**Pull request closed without merging**